projects/drivers/xilinx_platform/platform_drivers.c

```diff
@@ -259,7 +259,7 @@ int32_t spi_write_and_read(struct spi_desc *desc,
 			   XSPIPS_CLK_PHASE_1_OPTION : 0));
 
 	XSpiPs_SetSlaveSelect(&desc->instance,
-			      0xf & ~desc->chip_select);
+			      desc->chip_select);
```

Comment:
Since this change affects other projects too, they should be updated as well by the same commit.